### PR TITLE
Responsive Styles for Dashboard

### PR DESCRIPTION
**Pull request by corwinharrell**
screen shot 2018-08-29 at 10 13 14 pm

### CODE REVIEW

**Comment by corwinharrell** [comment]
This one was a doozy.

File: src/containers/Dashboard/Dashboard.module.scss [hunk 1]
```diff
@@ -18,52 +20,107 @@
...
+  width: 100%;
+
   @media only screen and (min-width: 48em) {
     display: flex;
     flex-direction: column;
     flex-grow: 1;
-    height: 27vh;
-    min-height: 25rem;
+    min-height: 0;
+    height: 30rem;
```
Comment (villanuevawill) [inline src/containers/Dashboard/Dashboard.module.scss]: should we use a variable here?

File: src/containers/Dashboard/index.js [hunk 2]
```diff
@@ -32,47 +32,38 @@ class DashboardComponent extends React.Component {
...
-            </div>
-          </div>
-          <div className="row center-xs">
-            <div className="col-xs-10">
-              <SubmissionsPanel
-                className={styles.submissionsPanel}
-                bodyClass={styles.bodyClass}
-              />
-            </div>
+        <div className={`pageWrapper-large ${styles.desktopContainer}`}>
```
Comment (villanuevawill) [inline src/containers/Dashboard/index.js]: <3 so much better without the rows and columns :)

File: src/containers/SubmissionsPanel/index.js [hunk 3]
```diff
@@ -69,11 +69,15 @@ class SubmissionsPanelComponent extends React.Component {
       <React.Fragment>
         <Table>
           <Table.Header>
-            <Table.HeaderCell flexGrow={4}>Bounty title</Table.HeaderCell>
-            <Table.HeaderCell flexGrow={3}>Fulfiller</Table.HeaderCell>
-            <Table.HeaderCell flexGrow={2}>Submission date</Table.HeaderCell>
-            <Table.HeaderCell flexGrow={2}>Status</Table.HeaderCell>
-            <Table.HeaderCell flexGrow={2}>Payment amount</Table.HeaderCell>
+            <Table.HeaderCell flexBasis="40%">Bounty title</Table.HeaderCell>
```
Comment (villanuevawill) [inline src/containers/SubmissionsPanel/index.js]: I love it :) % flexbasis here

Comment (villanuevawill) [reply src/containers/SubmissionsPanel/index.js]: @corwinharrell  my only Q here that confuses me - is some of these don't need to grow vs. the title as an example.

File: src/containers/UserStats/index.js [hunk 4]
```diff
@@ -40,46 +40,50 @@ class UserStatsComponent extends React.Component {
...
-              hash={public_address}
-              img={
-                is_profile_image_dirty
-                  ? ipfsToHttp(profileDirectoryHash, profileFileName)
-                  : profile_image
-              }
-              to={`/profile/${public_address}`}
-              className={styles.alignLeft}
-            />
-          </div>
```
Comment (villanuevawill) [inline src/containers/UserStats/index.js]: nice - removed the empty div and fluidity

File: src/explorer-components/SubmissionItem/index.js [hunk 5]
```diff
@@ -42,18 +42,26 @@ const SubmissionItem = props => {
...
-        {formattedTime}
+      <Table.Cell
+        flexBasis="15%"
+        contentType="numerical"
+        headerText="Submission date"
+      >
+        <Text typeScale="Body">{formattedTime}</Text>
       </Table.Cell>
-      <Table.Cell headerText="Status" flexGrow={2}>
+      <Table.Cell flexBasis="16%" headerText="Status">
```
Comment (villanuevawill) [inline src/explorer-components/SubmissionItem/index.js]: def. want to understand why flexBasis here and removal of grow.

File: src/components/Table/index.js [hunk 6]
```diff
@@ -6,21 +6,27 @@ import { Text } from 'components';
...
       </div>
     );
   }
 }
 
 HeaderCell.propTypes = {
   flexBasis: PropTypes.string,
-  flexGrow: PropTypes.number
+  flexGrow: PropTypes.number,
+  contentType: PropTypes.oneOf(['nonNumerical', 'numerical'])
```
[the remark this answers is not in the document]
Comment (corwinharrell) [reply src/components/Table/index.js]: @villanuevawill for numerical data in a table, it right-aligns the content of the cell.

File: src/containers/Dashboard/Dashboard.module.scss (hunk 1 above)
Comment (corwinharrell) [reply src/containers/Dashboard/Dashboard.module.scss]: Good catch, I'll go in and replace that.

**Pull request merged**